src/cli.c

```diff
@@ -0,0 +1,115 @@
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <limits.h>
+#include <getopt.h>
+#include <stdio.h>
+#include <errno.h>
+
+#include "cli.h"
+#include "logging.h"
+
+/******************************************************
+ *** LOCAL DEFINES                                  ***
+ ******************************************************/
+
+#define MAX_HZ (1000000L)
```

Comment:
```suggestion
```

Reply:
+1, accidental copy-paste?

Reply:
Yes, basically the same files as in cretro